index.html

```diff
@@ -1,29 +1,29 @@
 <!DOCTYPE html>
-<html lang=“ja”>
+<html lang="en">
   <head>
-    <meta charset=“UTF-8" />
-    <meta name=“viewport” content=“width=device-width, initial-scale=1.0" />
-    <meta http-equiv=“X-UA-Compatible” content=“ie=edge” />
-    <link rel=“stylesheet” href=“style.css” type=“text/css” />
-    <title>TerraceTechフロントエンドエンジニア養成所(I used $i）</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
   </head>
   <body>
-      <div>
-        <ul id="js-list"></ul>
-      </div>
+    <div>
+      <ul id="js-list"></ul>
+    </div>
   <script>   
     const ul = document.getElementById('js-list');
-      for(let i=1; i<=2; i++) {
-      const li = document.createElement('li');
-      ul.appendChild(li);
-      const a = document.createElement('a');
-      a.textContent = `a${i}`;
-      a.href= `a${i}.html`;
-      li.appendChild(a);
-      const img = document.createElement('img');
-      img.src = '/img/bookmark.png';
-      a.insertBefore(img, a.firstChild);
+    for(let i=1; i<=2; i++) {
+    const li = document.createElement('li');
+    ul.appendChild(li);
+    const a = document.createElement('a');
+    a.textContent = `a${i}`;
+    a.href= `a${i}.html`;
+    li.appendChild(a);
+    const img = document.createElement('img');
+    img.src = '/img/bookmark.png';
+    a.insertBefore(img, a.firstChild);
 }
   </script>
```

Comment:
I'm glad you noticed that backquotes and single quotes are similar, but not used in the same way.
It's so small that it's easy to misread.

It would be a good idea to separate the html file and the js file separately for future work.

Reply:
Thank you for your comments.
I will try to separate the html and js files separately.